src/main/java/frc/robot/subsystems/kicker/KickerSubsystem.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.subsystems.kicker;
+
+
+import org.littletonrobotics.junction.Logger;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+
+public class KickerSubsystem extends SubsystemBase {
+    KickerIO io;
+    KickerIOInputsAutoLogged inputs;
+
+
+    public KickerSubsystem(KickerIO io) {
+        this.io = io;
+        inputs =  new KickerIOInputsAutoLogged();
+
+        SmartDashboard.putData("reset Kicker value", new InstantCommand(
+            ()->{io.reset(0);}
+            ));
+    }
+
+    public InstantCommand reset (){
+       return new InstantCommand(
```

Comment:
can use `this.runOnce(() -> //code)` instead of `new InstantCommand`, which is slightly cleaner (this is me being pedantic, also i only learned about it after i taught yall how to do it this way so oops)